motioneye/webhook.py

```diff
@@ -40,12 +42,18 @@ def main(parser, args):
 
     logging.debug('hello!')
     logging.debug('method = %s' % options.method)
+    logging.debug('accept = %s' % options.accept)
+    logging.debug('useragent = %s' % options.useragent)
     logging.debug('url = %s' % options.url)
 
     headers = {}
     parts = urllib.parse.urlparse(options.url)
     url = options.url
     data = None
+    if 'accept' in options:
+        headers['Accept'] = options.accept
+    if 'useragent' in options:
+        headers['User-Agent'] = options.useragent
```

Comment:
Is it now skipped as well if those are empty strings?

Reply:
Pushed a change to make sure